package.json

```diff
@@ -104,35 +95,39 @@
 		"webpack": "3.10.0",
 		"webpack-rtl-plugin": "github:yoavf/webpack-rtl-plugin#develop"
 	},
+	"babel": {
+		"presets": [
+			"@wordpress/default"
+		],
+		"env": {
+			"gettext": {
+				"plugins": [
+					[
+						"./i18n/babel-plugin",
+						{
+							"output": "languages/gutenberg.pot"
+						}
+					]
+				]
+			}
+		}
+	},
 	"jest": {
 		"collectCoverageFrom": [
 			"(blocks|components|date|editor|element|i18n|data|utils|edit-post)/**/*.js"
 		],
-		"coveragePathIgnorePatterns": [
-			"<rootDir>/[^/]+/build/index.js"
-		],
-		"coverageDirectory": "coverage",
 		"moduleNameMapper": {
-			"\\.(scss|css)$": "<rootDir>/test/unit/style-mock.js",
 			"@wordpress\\/(blocks|components|date|editor|element|i18n|data|utils|edit-post)": "$1"
 		},
-		"modulePaths": [
-			"<rootDir>"
-		],
+		"preset": "@wordpress/jest-preset-default",
 		"setupFiles": [
-			"<rootDir>/test/unit/setup-globals.js",
+			"<rootDir>/test/unit/setup-blocks.js",
 			"<rootDir>/test/unit/setup-wp-aliases.js"
 		],
-		"setupTestFrameworkScriptFile": "<rootDir>/test/unit/setup-test-framework.js",
-		"testMatch": [
-			"<rootDir>/(blocks|components|date|editor|element|i18n|data|utils|edit-post)/**/test/*.js"
-		],
-		"timers": "fake",
 		"transform": {
 			"^.+\\.jsx?$": "babel-jest",
```

Comment:
@youknowriad it's back here. I had to reference `babel-jest` here because it doesn't merge options with preset. I opened and issue in Jest to fix it: https://github.com/facebook/jest/pull/5505.